packages/fxa-settings/src/pages/Signin/container.tsx

```diff
@@ -69,6 +74,7 @@ import { GqlKeyStretchUpgrade } from '../../lib/gql-key-stretch-upgrade';
 import { setCurrentAccount } from '../../lib/storage-utils';
 import { useCheckReactEmailFirst } from '../../lib/hooks';
 import { cachedSignIn } from './utils';
+import OAuthDataError from '../../components/OAuthDataError';
```

Comment:
Unused import

Reply:
Wait for the rebase once FXA-11297 lands.